data/software.json

```diff
@@ -2020,7 +1990,9 @@
         ],
         "doap": null,
         "name": "XMPP Status Checker",
-        "platforms": [],
+        "platforms": [
+            "Browser"
```

Comment:
I wonder if we need a 'project' category

Reply:
Instead of "platforms: Browser"?

Reply:
@Echolon what do you mean by that? And: is it in the scope of this PR?

Reply:
"Browser" is fine for this project.